design/layer2-prefer-node.md

```diff
@@ -0,0 +1,103 @@
+# Configuring Preferred Node To Be Layer2 Leader
+
+## Summary
```

Comment:
sorry to bother, would you mind filing this as a separate pr, independent from the implementation?

Reply:
Of course, the separate PR for design is created: https://github.com/metallb/metallb/pull/1804